### ISSUE TITLE
Recommended Trades Should Consider Pending Trades

### ISSUE THREAD

**Issue by controllerface** [controllerface/EDEPS#8]
The trades listed in the pending trades drop down does check the players' inventory to determine if the trade is possible, but currently doesn't also check if the material is "committed" to another trade. 

Because of this, a trade may show up as recommended, but then when added to the task list, the player will find they don't have enough material to complete that trade, because they've already allocated the material cost for another trade. 

Basically, when considering if the trade should be recommended or not, the sum of pending trades of the given material should be subtracted from the check amount. If a trade is filtered out this way, it should move to the "other trades" drop down, which is just a reference list.

**Comment by controllerface** [comment]
this is done
